verification/certifyKey.go

```diff
@@ -433,9 +433,11 @@ func checkCertificateStructure(t *testing.T, certBytes []byte) *x509.Certificate
 
 func testCertifyKey(d TestDPEInstance, client DPEClient, t *testing.T, simulation bool) {
 	ctx := getInitialContextHandle(d, client, t, simulation)
-	if simulation {
-		defer client.DestroyContext(ctx, 0)
-	}
+	defer func() {
+		if simulation {
+			client.DestroyContext(ctx, 0)
+		}
+	}()
```

Comment:
Fixed code here to place TagTCI testcase after CertifyKey_Simulation testcase in `AllTestCases` list. 

This is required because handle created in CertifyKey_Simulation mode interfered with derived context created for TagTCI tests with below error. 
```
=== RUN   TestRunAll/DefaultSupport-ExtendDerivedTciTestCase
    tagTCI.go:171: [FATAL]: Error while creating default child handle in default context: argument is invalid
```